demo/ios/README.md

```diff
@@ -1,15 +1,32 @@
 # Porcupine iOS Demo
 
```

Comment:
Add a short introductory paragraph explaining the differences between the demos. Either that or a sentence or two below each demo's title explaining the demo